cmake/external_llvm.cmake

```diff
@@ -25,8 +25,8 @@ endif()
 ExternalProject_Add(
     ext_clang
     PREFIX clang
-    GIT_REPOSITORY https://github.com/llvm-mirror/clang.git
-    GIT_TAG 26cac19a0d622afc91cd52a002921074bccc6a27
+    URL http://releases.llvm.org/5.0.2/cfe-5.0.2.src.tar.xz
```

Comment:
Please move these versions up to `NGRAPH_` variables at the top of the file like in the mkldnn and protobuf cmakes. This makes it easier for validation to go through SDL.